config.go

```diff
@@ -36,6 +41,8 @@ type TykPumpConfiguration struct {
 	HealthCheckEndpointName string                     `json:"health_check_endpoint_name"`
 	HealthCheckEndpointPort int                        `json:"health_check_endpoint_port"`
 	OmitDetailedRecording   bool                       `json:"omit_detailed_recording"`
+	base64DecodeRawData			bool											 `json:"base64_decode_raw_data"`
+	obfuscateAuthHeader		  ObfuscateAuthHeader
```

Comment:
@letzya I think we should change this or add a json tag in the `obfuscateAuthHeader` field.

If we do
```
base64DecodeRawData			bool											 `json:"base64_decode_raw_data"`
ObfuscateAuthHeader 
```
We could have `obfuscate_keys` and `auth_header_name` at the root config level.